### ISSUE TITLE
KeeFox will not install

### ISSUE THREAD

**Issue by deejaybags** [luckyrat/KeeFox#517]
KeeFox 1.4.8.1-signed will not install on FireFox 39.0 using using KeePass 2.29.
I click add to FireFox to install KeeFox, then restart FireFox as is requested. 
After restarting, the add-on screen keeps showing "KeeFox will be installed after you restart FireFox" and no button appears.

_edit to add more info_
I do not have a KeeFox toolbar button and the link chrome://keefox/content/install.xul does not work as shown in the troubleshooting guide.
Trying to follow the manual install steps, looking in my "profile"/"extensions" folder, there is no "keefox@chris.tomlinson" folder. 
There is a "staged" / "keefox@chris.tomlinson" / "deps" folder.
It does not contain a "KeePassRPC.plgx" file.
This folder contains "CheckForAdminRights.exe" , "KeeFoxElevate.exe"  and "KeePassRPCCopier.exe" file. 

Any help would be great, thanks.

**Comment by luckyrat** [comment]
Sounds like Firefox failed to stage the addon installation correctly. Maybe try deleting that staged folder and starting again? Or ask on the Firefox support forums for more general support relating to failed add-on installs (it doesn't look like it's a problem with KeeFox).

The only other thing that comes to mind is if you have some kind of external security software which is incorrectly deleting the KeePassRPC.plgx file when Firefox tries to stage the installation. (Maybe a dodgy anti-virus program?)

**Comment by deejaybags** [comment]
Hi Luckyrat, 

I think you are right about the anti-virus program doing something. I was using my work machine when I reported this problem yesterday. Last night I tried on my home machine and it worked perfectly.

I will try again with the work machine, and see if I can sort it out just in case someone else gets this problem also. 

Thanks for the suggestions and help. I'll close this issue off, but will report here on any findings re the work machine.
bags
